service/frontend/token.go

```diff
@@ -55,7 +56,7 @@ func validatePaginationToken(
 ) error {
 
 	execution := request.Execution
-	if request.GetNamespace() != token.GetNamespace() ||
+	if (request.GetNamespaceId() != token.GetNamespaceId() && request.GetNamespace() != token.GetNamespace()) ||
```

Comment:
probably should use `||` here ?

or this is due to compatibility?

Reply:
yes, this is for compatibility. I will remove `namespace` after next release. There is TODO for it in proto file.